client/my-sites/stats/stats-countries/index.jsx

```diff
@@ -77,71 +76,55 @@ module.exports = React.createClass( {
 			mapData.push( [ country.label, country.value ] );
 		} );
 
-		summaryPageLink = '/stats/' + this.props.period.period + '/countryviews/' + this.props.site.slug + '?startDate=' + this.props.date;
+		const summaryPageLink = '/stats/' + this.props.period.period + '/countryviews/' + this.props.site.slug + '?startDate=' + this.props.date;
+		const geochart = <Geochart data={ mapData } dataList={ this.props.dataList } />;
```

Comment:
I don't think we need to define these as `const`s here ( and countries below ), just have the component calls inline where they are being used.